db/data/seedDatabase.js

```diff
@@ -1,40 +1,103 @@
-const makeFakeRestaurant = require('./fakeDataGenerator.js');
+require('dotenv').config();
 const { MongoClient } = require('mongodb');
+const cluster = require('cluster');
+const numCPUs = require('os').cpus().length;
 
-const batchSize = parseInt(process.env.BATCH_SIZE, 10) || 10000;
-const numBatches = parseInt(process.env.NUM_BATCHES, 10) || 1000;
+const makeFakeRestaurant = require('./fakeDataGenerator.js');
 
-const url = 'mongodb://localhost:27017';
+const batchSize = parseInt(process.env.BATCH_SIZE, 10) || 10000;
+const numRecords = parseInt(process.env.NUM_RECORDS, 10) || 10000000;
 
-MongoClient.connect(url, (err, client) => {
+const connectToDb = async () => {
+  const url = 'mongodb://localhost:27017';
```

Comment:
minor note mostly just to keep in mind for later on: when it comes time to seed the db in deployment this might need to take the mongo address from an environment variable if you don't want to re-hardcode a new address for production environment. not critical now but just flagging for later!

Reply:
good idea, I moved it to an environment variable